src/App.tsx

```diff
@@ -1,47 +1,38 @@
-import HelpScout, { NOTIFICATION_TYPES } from "@helpscout/javascript-sdk";
+import { SidePanel, Sidebar } from './views';
 import {
-  Button,
-  DefaultStyle,
-  Heading,
-  useSetAppHeight,
-  Text,
-  useHelpScoutContext,
-} from "@helpscout/ui-kit";
-import { useEffect, useState } from "react";
+  BrowserRouter as Router,
+  Route,
+  createBrowserRouter,
+  createRoutesFromElements,
+  Outlet,
+  RouterProvider,
+} from 'react-router-dom';
+import { DefaultStyle } from '@helpscout/ui-kit';

```

Comment:
Should we move relative imports down?

Reply:
Nice catch! Should we install `@trivago/prettier-plugin-sort-imports` or just do it manually?

Reply:
I opted not to add any packages and instead manually formatted the imports in e1dc765f12b42c3ab70b88bc2638c0a05c344b3b. I'll go ahead and merge this :)